app/Traits/ApiResponser.php

```diff
@@ -91,16 +92,16 @@ protected function showMessage($message, $code = 200)
     protected function paginate(Collection $collection)
     {
         $rules = [
-            'per_page' => 'integer|min:2|max:50'
+            'limit' => 'integer|min:2|max:50'
         ];
         
         Validator::validate(request()->all(), $rules);
         
         $page = LengthAwarePaginator::resolveCurrentPage();
         
-        $prePage = 20;
-        if (request()->has('per_page')) {
-            $prePage = request()->per_page;
+        $prePage = $this->limit;
```

Comment:
key đã chuyển thành `limit` rồi thì tên biến đoạn này cũng phải đổi.
Có thể không cần khởi tạo biến mà dùng luôn `$this->limit` cũng được.